### DISCUSSION TITLE
Symbols

### DISCUSSION THREAD

**Discussion by alabotski** [iSoron/uhabits#1575] [Help & FAQ]
Can you tell me the difference between the labels

"+"
"-"
"×"
"✓" (filled) -
"✓" (not fill)

**Comment by alabotski** [comment]
![image](https://user-images.githubusercontent.com/439337/203507967-b6a0beb1-3b0b-4bd7-9d89-4a41d23f69a0.png)

**Comment by alabotski** [comment]
Can anyone help explain the difference between characters?
What is the difference

**Comment by alabotski** [comment]
![image](https://user-images.githubusercontent.com/439337/216888432-24edeb24-65e6-4abd-a871-de220a30e82f.png)
What is the difference between these labels?

**Reply by exanrauzer** [reply]
Just check settings of the app.

**Reply by alabotski** [reply]
What I should check?

**Reply by KosmicGR** [reply]
Here's a quick rundown:

- Filled Checkmark (✔): You completed this task today
- Outline of checkmark: This is for habits that aren't daily, it indicates that completion is optional for today (e.g. you completed a weekly task on a Monday, every day until the next Monday will have these)
- Cross (❌): This is a lapse, you didn't do this task today
- Dash (➖): This is a skip day, as explained in the screenshot below
- Question Mark (❔): This is a day that's missing data, as explained in the screenshot below

![Screenshot 2023-02-06 170425](https://user-images.githubusercontent.com/40884601/217007163-e0761aa4-79a3-4085-8213-83c2ec60b1ac.png)

This screenshot was taken in the settings menu.
Hope this helped.
